package.json

```diff
@@ -65,7 +66,7 @@
     "wdio-mocha-framework": "^0.3.1",
     "webdriverio": "4.0.9",
     "whatwg-fetch": "0.10.1",
-    "yuidocjs": "0.10.0",
-    "yuidoc-lucid-theme": "git+ssh://git@github.com/Shopify/yuidoc-lucid-theme.git#master"
+    "yuidoc-lucid-theme": "git+ssh://git@github.com/Shopify/yuidoc-lucid-theme.git#master",
```

Comment:
The `yui-doc-lucid-theme` dep should be set to a specific commit hash and not #master. So I think this should be:
```
"yuidoc-lucid-theme": "Shopify/yuidoc-lucid-theme#0b226d5"
```